src/robusta/integrations/mail/sender.py

```
@@ -84,6 +112,171 @@ def send_finding(self, finding: Finding, platform_enabled: bool, include_headers
+            msg['To'] = ', '.join(destination['ToAddresses'])
+            msg['Subject'] = message['Subject']['Data']
+            
+            # Add HTML and text bodies
+            msg.attach(MIMEText(message['Body']['Html']['Data'], 'html'))
+            msg.attach(MIMEText(message['Body']['Text']['Data'], 'plain'))
+            
+            # Add attachments
+            for file_block in file_blocks:
+                attachment = MIMEApplication(file_block.contents)
+                attachment.add_header('Content-Disposition', 'attachment', 
+                                    filename=file_block.filename)
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)